sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -500,6 +502,17 @@ public OutT apply(KafkaRecord<KeyT, ValueT> record) {
             return new KafkaConsumer<>(config);
           }
         };
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("topics", Joiner.on(",").join(getTopics())));
+      for (Map.Entry<String, Object> conf : getConsumerConfig().entrySet()) {
```

Comment:
Writing entire config might not be that useful. Is there a way to find out if a specific config is non-default?
You could use ConsumerConfig.originals(). Among those, the most important one could be at the top:
   `BOOTSTRAP_SERVERS_CONFIG "bootstrap.servers"`.

`RECEIVE_BUFFER_CONFIG` and `ENABLE_AUTO_COMMIT_CONFIG` could be skipped unless they differ from what we set them to above. (optional)